### ISSUE TITLE
Does not honor OptOut requests

### ISSUE THREAD

**Issue by Rosensteel** [Sujal-7/WikiSummarizerBot#5]
I've clicked the link to send OptOut messages dozens of times.  The bot does not honor the requests.

**Comment by schackartk** [comment]
Agreed. Looking at the code it is not apparent why. The place where opted-out usernamesa are stored is not public. Looking at the code, you should also receive a reply saying that you have been opted out. In the case that you are already opted out, you should get a reply saying so as well.

I get neither of these messages.

**Comment by 097115** [comment]
And you can't block this little shit, too.
